x-pack/plugins/lists/server/routes/import_exceptions_route.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { extname } from 'path';
+
+import { schema } from '@kbn/config-schema';
+import { transformError } from '@kbn/securitysolution-es-utils';
+import { importExceptionsResponseSchema } from '@kbn/securitysolution-io-ts-list-types';
+import { EXCEPTION_LIST_URL } from '@kbn/securitysolution-list-constants';
+import { validate } from '@kbn/securitysolution-io-ts-utils';
+import { ImportQuerySchemaDecoded, importQuerySchema } from '@kbn/securitysolution-io-ts-types';
+
+import type { ListsPluginRouter } from '../types';
+import { ConfigType } from '../config';
+
+import { buildRouteValidation, buildSiemResponse, getExceptionListClient } from './utils';
+
+/**
+ * Takes an ndjson file of exception lists and exception list items and
+ * imports them by either creating or updating lists/items given a clients
+ * choice to overwrite any matching lists
+ */
+export const importExceptionsRoute = (router: ListsPluginRouter, config: ConfigType): void => {
+  router.post(
+    {
+      options: {
+        body: {
+          maxBytes: config.maxImportPayloadBytes,
```

Comment:
Did you want `maxImportPayloadBytes` or `maxExceptionsImportSize` that you defined in the config?